mathics/core/parser/convert.py

```diff
@@ -114,6 +114,9 @@ def convert_Number(self, node: AST_Number) -> tuple:
                     prec10,
                 )
             else:
+                # "0`3"->0
```

Comment:
This comment is cryptic. The explanation given in the example  is clearer:, a relative uncertainty value times zero is zero.  

Personally, I stay away from words like more mysterious and complicated sounding words "vanish" when I want to convey something simple and definite like the product of zero and a number is zero. 

In research and academia you come across this feature where instead of saying something simply that everyone can easily understand, it is couched in more complicated words. This is why I think Richard Feynman was so popular - he could explain things in a simple manner.

Reply:
Actually, I left this because I supposed that there was a place in the documentation where this notation for absolute and relative errors was established, but no, it wasn't.

Reply:
Ok - I am sorry. But I'd appreciate it if would remove the crypticness in this code when it appears. Thanks.

Reply:
Now I improved the comments and added a new section in the documentation. Sadly, nice examples like

```
Pi == 3.1416`4
```
does not work by now...

Reply:
When you have time, please check the redaction of the new section in the documentation.